### PR TITLE
Tweak some images for loading and waiting

### PR DESCRIPTION
**Pull request by huayunh**
## Changes proposed in this Pull Request
- Update loading and waiting images using some gifs made possible by the latest design patterns Figma library.

## Screenshots
Some update samples:
progress-bar f56b489c

skeleton cb5026cc

### CODE REVIEW

File: src/docs/patterns/loading.mdx [hunk 1]
```diff
@@ -39,32 +40,36 @@ There are a number of different ways to indicate to a user that something is hap
...
+/>
+
+The skeleton should match as closely as possible the real _structure_ of the data, using shapes like rectangles and circles. The data _length_, however, may not be available at the time of loading, and it is likely that your skeleton ended up looking slightly different from your real data. This is OK; your mission is only to set up users' expectations.
+
+<ImageGrid
+    images={[Wave, Pulsate]}
+    caption={[
+        'Skeletons are usually animated, to indicate that the application is still running and responding. There are two kinds of skeleton animations.',
+        '1: Wave, where a flash glides pass gray shapes.',
+        '2: Pulsate, where the gray shades change their brightness periodically.',
```
Comment (joebochill) [inline src/docs/patterns/loading.mdx:66]:
```suggestion
        '1: Wave, where a gradient is animated from side to side over gray shapes.',
        '2: Pulsate, where the brightness of the gray shapes is animated periodically.',
```

**Review by joebochill — COMMENTED** [review]
- The first skeleton image has a weird slide-down effect on the cards at the end of the animation.

**Comment by huayunh** [comment]
> The first skeleton image has a weird slide-down effect on the cards at the end of the animation.

I was trying to experiment if there is a smoother way to show the beginning / end of a gif. How would you suggest me to redo it?

**Comment by joebochill** [comment]
> > The first skeleton image has a weird slide-down effect on the cards at the end of the animation.
> 
> I was trying to experiment if there is a smoother way to show the beginning / end of a gif. How would you suggest me to redo it?

Maybe just a longer delay on the 'loaded' content? It just seems weird for it to move.

**Review by joebochill — APPROVED** [review]

**Pull request merged**